packages/rancher-istio/1.19/rancher-kiali-server/generated-changes/patch/templates/deployment.yaml.patch

```diff
@@ -1,6 +1,16 @@
 --- charts-original/templates/deployment.yaml
 +++ charts/templates/deployment.yaml
-@@ -53,7 +53,7 @@
+@@ -6,6 +6,9 @@
+   namespace: {{ .Release.Namespace }}
+   labels:
+     {{- include "kiali-server.labels" . | nindent 4 }}
++  annotations:
++    "helm.sh/hook": post-install
```

Comment:
Do we need to add post-upgrade ?

Reply:
As discussed, it isn't needed since in the case of an upgrade the istiod deployment already exists